components/Ripple/BUILD

```diff
@@ -43,7 +43,7 @@ mdc_objc_library(
     name = "private",
     hdrs = native.glob(["src/private/*.h"]),
     includes = ["src/private"],
-    visibility = ["//visibility:private"],
+    visibility = ["//:__subpackages__"],
```

Comment:
Why is this necessary?

Reply:
This is part of the ThumbTrack Ripple Testing:
Checking if MDCRippleView has an activeRippleLayer (which is an instance of class MDCRippleLayer), helps identify if invoking `touchesBegan:touches` on the ThumbTrack initializes a ripple layer to show the ripple or not. To be able to check activeRippleLayer, I need to import MDCRippleLayer which is under private.
I could just expose MDCThumbTrack in the visibility, but i assumed __subpackages__ allows any future MaterialComponent to use this BUILD target to use it for testing.

Reply:
This visibility is a bit too open; it will allow any target in MDC make use of private implementation details. Can we restrict this to just the test target in some manner?

Reply:
I removed the dependency entirely, and used `@class MDCRippleLayer` instead. There was no need for me to get the entire class just to check nullability.

Reply:
💯